content/maxwell3_fdem/emi_tutorial/amplitudeandphase.rst

```diff
@@ -0,0 +1,129 @@
+.. _amplitudeandphase:
+
+Amplitude and phase
```

Comment:
I would give some thought to the title here - is it "Amplitude and Phase" or do you want to make it more specific to the fields? Just a thought.